hoss/dimension_utilities.py

```diff
@@ -55,6 +59,42 @@ def is_index_subset(message: Message) -> bool:
     )
 
 
+def get_override_projected_dimensions(
+    varinfo: VarInfoFromDmr,
+    override_variable_name: str,
+) -> str:
```

Comment:
) -> str | None

Reply:
Changes are in commit [756f7c0](https://github.com/nasa/harmony-opendap-subsetter/pull/15/commits/756f7c05d69fc135b5fc39825f5d7e1dde5daada)

Reply:
@autydp said it is resolved